src/ui/content-settings/__tests__/ContentSettingsCropRatiosContainer.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+
+import { configEnzymeAdapter } from 'testutils/helpers';
+import ContentSettingsCropRatiosContainer from 'ui/content-settings/ContentSettingsCropRatiosContainer';
+import ContentSettingsCropRatios from 'ui/content-settings/ContentSettingsCropRatios';
+
+configEnzymeAdapter();
+
+jest.mock(
+  'ui/content-settings/ContentSettingsCropRatios',
+  () => jest.fn(() => <div>ContentSettingsCropRatios</div>),
+);
+
+const initialState = {
+  apps: {
+    cms: {
+      contentSettings: {
+        cropRatios: [
+          '4:9',
+          '16:9',
+        ],
+      },
+    },
+  },
+};
+
+const store = configureMockStore([thunk])(initialState);
+
+const actionList = [
+  'onAdd',
+  'onDelete',
+  'onUpdate',
+];
+
+describe('ContentSettingsCropRatiosContainer', () => {
+  it('should render ContentSettingsCropRatios with correct props', () => {
+    shallow(
+      <ContentSettingsCropRatiosContainer store={store} />,
```

Comment:
Hi @raxkaynan. I tried running unit test and it seems there's an error in this area. Please check.

```
src/ui/content-settings/__tests__/ContentSettingsCropRatiosContainer.test.js
  ● ContentSettingsCropRatiosContainer › should render ContentSettingsCropRatios with correct props

    Invariant Violation: [React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry.

      38 | describe('ContentSettingsCropRatiosContainer', () => {
      39 |   it('should render ContentSettingsCropRatios with correct props', () => {
    > 40 |     shallow(
         |     ^
      41 |       <ContentSettingsCropRatiosContainer store={store} />,
      42 |     ).dive().dive();
      43 | 

      at invariant (node_modules/invariant/invariant.js:40:15)
      at invariantIntlContext (node_modules/react-intl/lib/index.js:473:3)
      at new InjectIntl (node_modules/react-intl/lib/index.js:557:7)
      at ReactShallowRenderer.render (node_modules/react-test-renderer/cjs/react-test-renderer-shallow.development.js:723:26)
      at render (node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:549:53)
      at fn (node_modules/enzyme-adapter-utils/src/Utils.js:99:18)
      at Object.render (node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:549:18)
      at new ShallowWrapper (node_modules/enzyme/build/ShallowWrapper.js:265:22)
      at shallow (node_modules/enzyme/build/shallow.js:21:10)
      at Object.it (src/ui/content-settings/__tests__/ContentSettingsCropRatiosContainer.test.js:40:5)
```

using `testutils/helpers.js > mockRenderWithIntl` could eliminate this error.